src/component/room/_room.scss

```diff
@@ -1,15 +1,60 @@
-.close-save {
+@import '_vars';
+
+// * {
+//   border: solid red .25px;
+// }
+
+.info-button-container {
+  display: grid;
+  grid-template-columns: repeat(4, 25%);
+  grid-template-rows: repeat(4, 25%);
+}
+
+.active-voters {
+  grid-row: 1 / span 1; 
+  grid-column: 1 / span 2;
+  align-self: center;
+  line-height: 36px;
   text-align: center;
-  position: relative;
+  margin: $default-button-margin;
+  font-size: 1.15em;
+  font-family: $font-main;
+  @media only screen and (min-width: 600px) {
+    grid-column: 1 / span 1;
+  }
+}
+
+.close-save-button {
+  align-self: center;
```

Comment:
What is this? Haven't seen it before. Teach me!

Reply:
haha can't in a comment!